src/node/node_to_node.h

```diff
@@ -64,6 +64,28 @@ namespace ccf
       return t;
     }
 
+    template <class T>
+    const T& recv_authenticated_with_load(const uint8_t*& data, size_t& size)
```

Comment:
I don't understand why this is necessary, can't this just be another message type?

Reply:
As discussed, let's have the seqno and view in the header, and the rest in the payload? That'll faciliate testing.

Reply:
discussed on a call with the group